src/views/css-styles/Default.jsx

```
@@ -0,0 +1,64 @@
+            <span>
+            {":default is a pseudo-class that represents an default element in a group of elements in the same context."}
+            {"However, the most of modern browsers uses !important for Default styles so you may find some style properties are not working."}
+            {"In this demo the background color of the default element is blue and the selected text color is red."}
+            </span>
+          </p>
+        </Card>
+        <Card>
+          <h2>:default check box</h2>
+          <form onSubmit={(e) => e.preventDefault()}>
+              <label>
+                <input type="checkbox" name="my-check" class="peer" checked/>
```

ESLint

[error] Unknown property 'class' found, use 'className' instead

(react/no-unknown-property)